packages/rest-hooks/src/resource/SimpleRecord.ts

```diff
@@ -8,6 +8,13 @@ interface SimpleResourceMembers<T extends typeof SimpleRecord> {
 
 /** Immutable record that keeps track of which members are defined vs defaults. */
 export default abstract class SimpleRecord {
+  // a 'unique' identifier to make referential equality comparisons easy
+  declare readonly _unq: string;
```

Comment:
this is sort of a hack for react not wanting variable sync comparisons. we'll be serializing this later